vars/githubscm.groovy

```diff
@@ -117,21 +117,38 @@ def forkRepo(String credentialID = 'kie-ci') {
 }
 
 def createPR(String pullRequestTitle, String pullRequestBody = '', String targetBranch = 'master', String credentialID = 'kie-ci') {
+    def pullRequestLink
+    try {
+        pullRequestLink = executeHub("hub pull-request -m '${pullRequestTitle}' -m '${pullRequestBody}' -b '${targetBranch}'", credentialID)
+    } catch (Exception e) {
+        println "[ERROR] Unable to create PR. Please make sure the targetBranch ${targetBranch} is correct."
+        throw e;
+    }
```

Comment:
the try/catch could be in the `executeHub` method I guess

Reply:
@radtriste I leave `executeHub` method in the more generic way possible and it's up to the method calling to deal with the exception thrown by the method. I guess is more helpful the `[ERROR] Unable to create PR. Please make sure...` message than something like `[ERROR] Error executing hub...`
wdyt?

Reply:
Alternatively, if we want to avoid duplicate try-catch, we can have a more general `createPR` method that accepts labels and call it from the simpler, overloaded, method without labels.

Reply:
the only problem I see about it @rsynek is every method we provide is exposed publicly to the world, I mean every method is part of the githubscm's "API". Additionally default arguments somehow creates problems when you try to overload them. wdyt guys?

Reply:
I don't know the exact limitations of overloading methods with default arguments; I was hoping that Groovy could match the call properly if the mandatory arguments of overloaded methods differ. If it's not the case, I guess the existing solution is good enough.